virtcontainers/sandbox.go

```diff
@@ -70,6 +70,9 @@ type State struct {
 
 	// PCI slot at which the block device backing the container rootfs is attached.
 	RootfsPCIAddr string `json:"rootfsPCIAddr"`
+
+	// Pid is the process id of the container process
```

Comment:
Could you elaborate a bit here. The comment describing `Pid` is not verbose enough IMO.